src/signaturevalidator.js

```diff
@@ -292,6 +292,7 @@ const createSignatureValidator = function(options) {
           port: req.port,
           resource: req.resource,
           method: req.method.toUpperCase(),
+          sourceIp: req.sourceIp,
```

Comment:
For consistency, this should probably have `|| '0.0.0.0'` so that we always get a valid IP in this slot (but a very distinctive one for cases where we do not know).